src/pages/ResourceForm.tsx

```diff
@@ -90,9 +96,31 @@ const ResourceForm: FC<ResourceFormProps> = ({ uppy, resource, resourceType }) =
       features: Yup.object().shape({
         dlr_title: Yup.string().required(t('feedback.required_field')),
       }),
+      licences: Yup.array().of(
+        Yup.object().shape({
+          identifier: Yup.string().required(t('feedback.required_field')).min(1, t('feedback.required_field')),
+        })
+      ),
     }),
   });
 
+  useEffect(() => {
+    async function getAllLicences() {
+      setIsLoadingLicenses(true);
+      setLoadingLicensesErrorStatus(StatusCode.ACCEPTED);
```

Comment:
Burde egentlig ikke denne settes til status 200 istedet for 202?

Reply:
derfor vi trenger å endre litt på denne ja :-)  Tenker vi kun bryr oss om error-codes og har en tom eller undefined string-verdi som default